libraries/chain/include/eosio/chain/genesis_state.hpp

```diff
@@ -16,25 +16,25 @@ struct genesis_state {
-      .max_block_net_usage                  = config::default_max_block_net_usage,
-      .target_block_net_usage_pct           = config::default_target_block_net_usage_pct,
-      .max_transaction_net_usage            = config::default_max_transaction_net_usage,
-      .base_per_transaction_net_usage       = config::default_base_per_transaction_net_usage,
-      .net_usage_leeway                     = config::default_net_usage_leeway,
-      .context_free_discount_net_usage_num  = config::default_context_free_discount_net_usage_num,
-      .context_free_discount_net_usage_den  = config::default_context_free_discount_net_usage_den,
-
-      .max_block_cpu_usage                  = config::default_max_block_cpu_usage,
-      .target_block_cpu_usage_pct           = config::default_target_block_cpu_usage_pct,
-      .max_transaction_cpu_usage            = config::default_max_transaction_cpu_usage,
-      .min_transaction_cpu_usage            = config::default_min_transaction_cpu_usage,
-
-      .max_transaction_lifetime             = config::default_max_trx_lifetime,
-      .deferred_trx_expiration_window       = config::default_deferred_trx_expiration_window,
-      .max_transaction_delay                = config::default_max_trx_delay,
-      .max_inline_action_size               = config::default_max_inline_action_size,
-      .max_inline_action_depth              = config::default_max_inline_action_depth,
-      .max_authority_depth                  = config::default_max_auth_depth,
+      .max_block_net_usage                 = config::default_max_block_net_usage,
+      .target_block_net_usage_pct          = config::default_target_block_net_usage_pct,
+      .max_transaction_net_usage           = config::default_max_transaction_net_usage,
+      .base_per_transaction_net_usage      = config::default_base_per_transaction_net_usage,
+      .net_usage_leeway                    = config::default_net_usage_leeway,
+      .context_free_discount_net_usage_num = config::default_context_free_discount_net_usage_num,
+      .context_free_discount_net_usage_den = config::default_context_free_discount_net_usage_den,
+
+      .max_block_cpu_usage                 = config::default_max_block_cpu_usage,
+      .target_block_cpu_usage_pct          = config::default_target_block_cpu_usage_pct,
+      .max_transaction_cpu_usage           = config::default_max_transaction_cpu_usage,
+      .min_transaction_cpu_usage           = config::default_min_transaction_cpu_usage,
+
+      .max_transaction_lifetime            = config::default_max_trx_lifetime,
+      .deferred_trx_expiration_window      = config::default_deferred_trx_expiration_window,
+      .max_transaction_delay               = config::default_max_trx_delay,
+      .max_inline_action_size              = config::default_max_inline_action_size,
+      .max_inline_action_depth             = config::default_max_inline_action_depth,
+      .max_authority_depth                 = config::default_max_auth_depth,
```

Comment:
Can we revert these whitespace changes as well.

Reply:
Done.